Source/BugsnagNotifier.m

```diff
@@ -506,20 +502,14 @@ - (void)internalClientNotify:(NSException *_Nonnull)exception
                               severity:BSGParseSeverity(severity)
                              attrValue:logLevel];
 
-    [self notify:exception.name ?: NSStringFromClass([exception class])
-             message:exception.reason
-        handledState:state
-               block:^(BugsnagCrashReport *_Nonnull report) {
-                 if (block) {
-                     block(report);
-                 }
-               }];
+    [self notify:exception handledState:state block:block];
```

Comment:
Would this alter the depth of the stack, as there's no longer an extra block?

Reply:
Yes, technically, however for how we use it, no. This function is only used in cocoa apps in react native, where a custom stacktrace is attached, disregarding depth altogether. And as it isn't a public interface, there should be no other effects.